config/settings.yml

```diff
@@ -13,4 +13,6 @@ plugins:
   slack_icon_url:
     default: ''
   slack_access_token:
-    default: ''
\ No newline at end of file
+    default: ''
+  post_to_slack_window_mins:
```

Comment:
After some internal discussion, let's change this to default 20 seconds instead.